packages/infrastructure/kube_deployment/vars.tf

```diff
@@ -91,6 +91,7 @@ variable "containers" {
     image              = string
     version            = string
     command            = list(string)
+    workingDir         = optional(string, "")
```

Comment:
This should be `optional(string, null)`. Unless I am mistaken, I have designed the module so that any `null` value will be stripped from the manifest prior to being submitted to the kubernetes provider.

Reply:
Let me test that before I re-request your review :)